backend/packages/Upgrade/src/auth/authorizationChecker.ts

```diff
@@ -1,12 +1,11 @@
 import { Action } from 'routing-controllers';
 import { Container } from 'typedi';
-
-import { Logger } from '../lib/logger';
 import { AuthService } from './AuthService';
 import { env } from '../env';
+import { UpgradeLogger } from '../lib/logger/UpgradeLogger';
 
 export function authorizationChecker(): (action: Action, roles: any[]) => Promise<boolean> | boolean {
-  const log = new Logger(__filename);
+  const log = new UpgradeLogger();
```

Comment:
Can we get logger instance from the request object here instead of creating our own logger?